src/constants/layers-urls.js

```diff
@@ -238,7 +238,7 @@ export const LAYERS_URLS = {
   // Vertebrate species modal
 
   [SPECIES_LIST]:
-    'https://utility.arcgis.com/usrsvcs/servers/5df959008b444568836d12a3ba215a90/rest/services/NRC_marine_species_data_20220323/FeatureServer',
+    'https://utility.arcgis.com/usrsvcs/servers/04986e0b667c4ad29539683d6ba2314f/rest/services/NRC_species_data_20200817_formatted/FeatureServer',
```

Comment:
If we are having problems with the land table please include also the feature tag and show keep the old one if we don't have REACT_APP_FEATURE_MARINE active